style.css

```diff
@@ -875,3 +880,515 @@ ul {
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 85%;
+    margin: 5% 0 0 5%;
+    border-radius: 24px;
+    background-color: #e5e5e5;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid #dfe1e6;
+    margin-top: 5%;
+    height: 60vh;
+    width: 100%;
+    border-radius: 24px;
+    margin-bottom: 5%;
+  }
+
+  .card-container img {
+    width: 50%;
+    height: 100%;
+    border-radius: 8px;
+  }
+
+  .faha2 {
+    order: 2;
+  }
+
+  .work-fonony {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-left: 8rem;
+    width: 50%;
+    margin-top: 10rem;
+  }
+
+  .card-tittle {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  .frame2-img {
+    margin-top: -1rem;
+    margin-left: -5.7rem;
+  }
+
+  .frame3-img {
+    display: flex;
+    align-items: center;
+  }
+
+  .project-info {
+    list-style: none;
+    display: flex;
+    width: 66%;
+    font-size: 16px;
+    margin-top: -5rem;
+    font-weight: bold;
+    flex-direction: row;
+    gap: 8px;
+    order: 2;
+    margin-left: -5.5rem;
+  }
+
+  ul.project-info li {
+    list-style-type: none;
+    position: relative;
+    white-space: nowrap;
+  }
+
+  ul.project-info li:not(:first-child)::before {
+    content: "\2022";
+    font-size: 40px;
+    position: absolute;
+  }
+
+  .fb {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #3b5998;
+  }
+
+  .full {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .year {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  #fb-title {
+    margin-top: -15rem;
+    margin-left: -5rem;
+  }
+
+  #fb-info {
+    list-style: none;
+    display: flex;
+    margin: 0;
+    width: 60%;
+    font-size: 16px;
+    margin-left: -5rem;
+    margin-top: -6rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    gap: 8px;
+    order: 2;
+  }
+
+  #back-list {
+    margin-left: -5.3rem;
+    margin-top: -11rem;
+  }
+
+  #fd-buttons {
+    margin-left: -5.4rem;
+    background-color: #396df2;
+    color: #dfe1e6;
+  }
+
+  #fd-buttons:hover {
+    background-color: #dfe1e6;
+    color: #396df2;
+  }
+
+  #description-fb {
+    display: flex;
+    margin-top: -4rem;
+    width: 100%;
+    margin-left: -5.3rem;
+  }
+
+  .description {
+    display: flex;
+    margin-top: -5rem;
+    width: 100%;
+    margin-left: -5.3rem;
+  }
+
+  ul.languages-list li {
+    white-space: nowrap;
+  }
+
+  /* i tried to fix space between description and languages-lis but it doesn't work */
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -9.5rem;
+  }
+
+  .buttons {
+    margin-left: -5.4rem;
+    margin-top: 3rem;
+    background-color: #dfe1e6;
+    color: #396df2;
+  }
```

Comment:
- [ ] As requested by the previous reviewer please make sure you follow the given Figma design when implementing the styling for the button as the background color should be `#ffffff` and not `#dfe1e6`. ✔
- Figma design:
  ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/236435820-c9f8fe54-6ca2-4e2d-b2f3-13a47f7bae67.png)